t/lib/t/MusicBrainz/Server/Validation.pm

```diff
@@ -187,10 +199,6 @@ test 'Test is_valid_partial_date' => sub {
     };
 };
 
-test 'Test encode_entities' => sub {
-    is(encode_entities('ãñ><"\'&Å'), 'ãñ&gt;&lt;&quot;&#39;&amp;Å');
```

Comment:
With the other `encode_entities`, this expands to `&atilde;&ntilde;&gt;&lt;&quot;&#39;&amp;&Aring;`.